apps/nacl_demos/Makefile

```diff
@@ -36,10 +36,10 @@ LDFLAGS_PNACL := -L $(LIBDIR_PNACL) -lppapi_cpp -lppapi
 DEMOS = game_of_life julia reaction_diffusion reaction_diffusion_2
 DEMO_OBJECTS = $(DEMOS:%=%_init) $(DEMOS:%=%_update) $(DEMOS:%=%_render)
 
-OBJECTS_32 = nacl_demos_32.o $(DEMO_OBJECTS:%=build_32/%.o)
-OBJECTS_ARM = nacl_demos_arm.o $(DEMO_OBJECTS:%=build_arm/%.o)
-OBJECTS_64 = nacl_demos_64.o $(DEMO_OBJECTS:%=build_64/%.o)
-OBJECTS_PNACL = nacl_demos_pnacl.bc $(DEMO_OBJECTS:%=build_pnacl/%.bc)
+OBJECTS_32 = nacl_demos_32.o $(DEMO_OBJECTS:%=build_32/%.a)
+OBJECTS_ARM = nacl_demos_arm.o $(DEMO_OBJECTS:%=build_arm/%.a)
+OBJECTS_64 = nacl_demos_64.o $(DEMO_OBJECTS:%=build_64/%.a)
+OBJECTS_PNACL = nacl_demos_pnacl.bc $(DEMO_OBJECTS:%=build_pnacl/%.a)
```

Comment:
re: .bc -> .a, that's correct: bitcode is ar'ed together for static libraries like other formats.

Re: manage to test this change; unfortunately, not yet, as our current build assumes Pepper 41, which apparently isn't readily available anymore from the naclsdk installer (Pepper 44 is the minimum provided). Not clear yet whether Pepper 41 is still available from a reliable source. (I experimentally just tried modifying to use Pepper 44 instead, but it fails at ABI verification time, which may be a legit issue or may just be a result of the mismatch between our pnacl-llvm and naclsdk versions.)